### ISSUE TITLE
Spawn with each item an additional non-physical coordinate system object

### ISSUE THREAD

**Issue by gaya-** [cram2/cram#74]
Items are spawned here:
``cram_3d_world/cram_bullet_reasoning/src/items.lisp``

Objects that don't cause collisions are spawned here:
``cram_3d_world/cram_bullet_reasoning/src/debug-window.lisp``

**Comment by gaya-** [comment]
Otherwise can always do:

```lisp 
(btr:add-objects-to-mesh-list "cram_bullet_reasoning" :directory "resource" :extension "stl")
(btr-utils:spawn-object  :axis-1 :axes :pose '((1.4 0.8 0.9) (0 0 0 1) ))
```

**Comment by gaya-** [comment]
```
(spawn-axis cl-tf-pose)
```

```
(defun spawn-axis (btr-object-name)
           (let ((pose (btr:pose btr-object))
                  (spawn-axis pose)))
```

**Comment by gaya-** [comment]
Solved in #93
